lib/routes/deepin/thread.ts

```diff
@@ -0,0 +1,92 @@
+import { Route } from '@/types';
+import ofetch from '@/utils/ofetch';
+import cache from '@/utils/cache';
+import { parseDate } from '@/utils/parse-date';
+
+export const route: Route = {
+    path: '/threads/:type',
```

Comment:
`type` should be [optional](https://hono.dev/docs/api/routing#optional-parameter) according to https://github.com/DIYgod/RSSHub/blob/b9b2e31821c51606f7e0b2421c9ec2b4b037c68a/lib/routes/deepin/thread.ts#L56